src/Manager/JwtManager.php

```diff
@@ -94,6 +94,13 @@ public function getJwtToken()
 
         if ($expiresIn) {
             $expiration = new \DateTime('now + ' . $expiresIn . ' seconds');
+        } elseif (count($jwtParts = explode('.', $body[$this->options['token_key']])) === 3
```

Comment:
maybe you could do one PR with the two features.

Reply:
What do you mean? Merge #8 and #9 in one PR?

Reply:
ok never mind, i see that the changes from #8 are in this one, so we can merge this one for all new feature. A little of doc in the readme would be nice :)

Reply:
I needed the change locally, and wasn't sure you would approve both :).

I wil update the readme!